store/src/state.rs

```diff
@@ -507,7 +507,7 @@ pub fn build_note_tree(notes: &[NoteCreated]) -> Result<BlockNoteTree, ApplyBloc
             note_type,
             note.tag
                 .try_into()
-                .map_err(|_| NoteError::InconsistentNoteTag(note_type, note.tag))?,
+                .map_err(|_| NoteError::InconsistentNoteTag(note_type, note.tag.into()))?,
```

Comment:
Would probably be better to change the error variant to take `u32` as the second value in a tuple.

Reply:
This is on `miden-base`. I can make the change though.

Reply:
Ah - in that case, it's fine to keep as is.